cognite/client/_api/data_modeling/instances.py

```diff
@@ -101,7 +101,7 @@ def load(
     ) -> _NodeOrEdgeList:
         resource_list = json.loads(resource_list) if isinstance(resource_list, str) else resource_list
         resources: list[Node | Edge] = [
-            Node.load(data) if data["instanceType"] == "node" else Edge.load(data) for data in resource_list
+            Node._load(data) if data["instanceType"] == "node" else Edge._load(data) for data in resource_list
```

Comment:
All these changes from `load` to `_load` shouldn't be necessary, right? We can still use the public method here?

Reply:
Yes, we can, it is an optimization. This is an internal method and we now that we have a dict, then we save a function call in the callstack by calling _load directly.